docs/guides/registries/self-signed.md

```diff
@@ -56,6 +55,15 @@ build:
     SAVE IMAGE --push --insecure <registry-hostname>/hello-earthly:with-love
 ```
 
+{% hint style='danger' %}
+##### Note
+
+The `http` and `insecure` settings are mutually exclusive. Setting `insecure=ture` should only be used when the registry is https, and you deliberately want to accept invalid or unverifiable certificates.
```

Comment:
```suggestion
The `http` and `insecure` settings are mutually exclusive. Setting `insecure=true` should only be used when the registry is https, and you deliberately want to accept invalid or unverifiable certificates.
```

Reply:
I also think that "...the registry is https, and you deliberately..." is a bit ... ambiguous maybe? saying `...the registry is https _and_ you deliberately..." seems to read more clearly to me - the comma is perhaps a misleading separator in this context.

Reply:
good point -- I simplified it (and also fixed the typo).